Combobox.js

```diff
@@ -272,7 +305,8 @@ define([
 			}
 		},
 
-		refreshRendering: function (oldValues) {
+		/* jshint maxcomplexity: 17 */
+		refreshRendering: function (oldValues, justCreated) {
 			var updateReadOnly = false;
 			if ("list" in oldValues) {
 				this._initList();
```

Comment:
It seems like a trick question... we only want to call `_initList()` once for a list, but [in the unlikely event that] `this.list` changes then we should call `_initList()` again.

Probably though line 311 should read `if ("list" in oldValues && this.list)` to prevent calling `_initList()` when `this.list` is null.

I did look at `prepareInput()` but I don't understand the connection.